api/v1-gcode/writeGcode.js

```diff
@@ -15,10 +14,7 @@ export const prepareGcode = (shape, name, tool, options = {}) => {
   let index = 0;
   const entries = [];
   for (const entry of ensurePages(shape.toKeptGeometry())) {
-    const op = toGcode(entry, tool, {
-      definitions: getDefinitions(),
-      ...options,
-    }).catch(getPendingErrorHandler());
+    const op = toGcode(entry, tool, options).catch(getPendingErrorHandler());
```

Comment:
The change to these is that options is now an object? Like {option1: true}

Reply:
Ah, it already was -- the difference is that we aren't mixing definitions into it anymore.